module/strategy/Defend/src/Defend.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef MODULE_STRATEGY_DEFEND_HPP
+#define MODULE_STRATEGY_DEFEND_HPP
+
+#include <Eigen/Core>
+#include <nuclear>
+
+#include "extension/Behaviour.hpp"
+
+namespace module::strategy {
+
+    class Defend : public ::extension::behaviour::BehaviourReactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+
+            /// @brief Region to defend the ball
+            Eigen::Vector4f defending_region = Eigen::Vector4f::Zero();
+
+        } cfg;
+
+        // Robot - Defending position on field
+        Eigen::Vector3f rDFf = Eigen::Vector3f::Zero();
```

Comment:
this also looks like a temporary variable? why is it a class variable?

Reply:
Removed it from being class memeber